recipes/xtl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xtl" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "97cb01feb042ee59727ea1dcf502ed300c3533f009a6e606094b2811c66ee1f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/QuantStack/xtl/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/xtl  # [unix]
+    - test -f ${PREFIX}/include/xtl/xbasic_fixed_string.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xtl/xtlConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/xtl/xtlConfigVersion.cmake  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\xtl\xbasic_fixed_string.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xtlConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\xtlConfigVersion.cmake (exit 0) else (exit 1)  # [win]
+
+about:
+  home: http://quantstack.net
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'The QuantStack tools library'
+  description: 'Basic tools (containers, algorithms) used by other packages from Quantstack'
+  doc_url: http://xtl.readthedocs.io
+  dev_url: https://github.com/QuantStack/xtl
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
```

Comment:
@SylvainCorlay please ACK :)

Reply:
![17mu35](https://user-images.githubusercontent.com/2397974/30421558-6872809e-993d-11e7-914a-a50a435990c7.jpg)

Reply:
Next time I'll think twice before adding you as a maintainer